sys/kern/src/descs.rs

```diff
@@ -128,7 +128,6 @@ impl RegionDesc {
 
         ratts.contains(RegionAttributes::WRITE)
             && !ratts.contains(RegionAttributes::DEVICE)
-            && !ratts.contains(RegionAttributes::DMA)
```

Comment:
This is going to be a problem for `dump_agent`: the extern SRAM regions are mapped as `READ | WRITE | DMA`, so if `dump_agent` crashes or we request a task dump, it will dump the dump regions into themselves.

I tested this by killing `dump_agent` with `jefe`.  Sure enough, this task dump is _too big_:
```
AREA TASK                  TIME       SIZE
   0 dump_agent            6200       225736
```

Previously, it was much more reasonable:
```
AREA TASK                  TIME       SIZE
   0 dump_agent            4900       4664
```

Reply:
Fixed this by having Jefe refuse to dump anything that is itself in the dump areas.